docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-2.md

```diff
@@ -17,8 +17,8 @@ management workspace (WMW).  Start by creating a WMW for the common
 workload, with the following commands.
 
 ```shell
-IN_CLUSTER=false SPACE_MANAGER_KUBECONFIG=$SM_CONFIG kubectl kubestellar ensure wmw wmw-c
-wmw_c_space_config=$PWD/temp-space-config/spaceprovider-default-wmw-c
+IN_CLUSTER=false kubectl kubestellar ensure wmw wmw-c
+WMW_C_SPACE_CONFIG=$PWD/temp-space-config/spaceprovider-default-wmw-c
```

Comment:
I think it would be a better design if the caller of `kubectl kubestellar ensure wmw` specified the output filename.

With this markdown in charge of the output filename, it can be much shorter and more convenient.